furnace/crossref_paper.py

```
@@ -1,9 +1,21 @@
-from local_deprecated.Reference import get_paper_info_from_REST
+from scholarly import scholarly
+# OPENAI SETUP
+import openai
+import requests
+import pandas as pd
+import matplotlib.pyplot as plt
+import string
+
+from furnace.Author import Author
+from furnace.Publication import Document
+from tools.Reference import filter_punctuation, get_paper_info_from_REST
```

Ruff (0.8.2)

`tools.Reference.filter_punctuation` imported but unused

Remove unused import: `tools.Reference.filter_punctuation`

(F401)